org.uqbar.project.wollok.lib/src/wollok/lang.wlk

```diff
@@ -1281,7 +1281,11 @@ class String {
 	 * 		"become".contains("CO")      ==> Answers false
 	 */
 	method contains(other) {
-		return self.indexOf(other) > 0
+		try {
+			return self.indexOf(other) >= 0
+		} catch e {
```

Comment:
I'd like to find a clean way of doing it in pure Wollok, but I think we can switch to use a native contains from WString temporarily. The wrapped String already knows contains(...).